test_game.py

```diff
@@ -1,45 +1,2 @@
-import unittest
-from game import Game
+""" to be added in a future ticket """
 
```

Comment:
If your tests are passing with the changes you made in the game class, you want to keep the tests. If they don't pass, then you want to refactor them in the current PR. This is a good technique to keep all changes relating to tests and code in the same PR so you can "refactor on green". Right now, the PR changes Game logic and is not backed by tests, which would not normally get approved by developers if you were on a team because this removes confidence in the code changes made - the tests show that the logic is sound, if that makes sense.

Reply:
I think Rani may already be gone for the day so I'll respond so you aren't blocked from moving forward. How would you feel about keeping the tests that you already wrote and then refactoring the tests and your source code when you pull the "Test Game" card? 

(I would also personally prefer if you removed the changes to the Game class that you made in this PR and copied them over to the feature branch for "Test Game" -- but if the other mentors aren't going to require that, it's ok with me if you leave them this time. But let's try to get in the habit of only making changes within the scope of a ticket's acceptance criteria!)